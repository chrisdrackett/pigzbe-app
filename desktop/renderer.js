import {AppRegistry} from 'react-native';
import App from './bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
