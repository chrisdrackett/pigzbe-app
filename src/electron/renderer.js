import {AppRegistry} from 'react-native';
// import App from '../../App';
import App from '../../dist/bundle.web.js';

AppRegistry.registerComponent('App', () => App);
AppRegistry.runApplication('App', {rootTag: document.getElementById('react-root')});
