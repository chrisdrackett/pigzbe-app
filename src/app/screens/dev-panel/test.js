import React from 'react';
import renderer from 'react-test-renderer';
import {DevPanelComponent} from './';

describe('DevPanel', () => {
    test('renders correctly', () => {
        renderer.create(<DevPanelComponent/>);
    });
});
