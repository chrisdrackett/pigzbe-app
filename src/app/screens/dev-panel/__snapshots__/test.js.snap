// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DevPanel renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-between",
      "left": 0,
      "paddingLeft": "6.25%",
      "paddingRight": "6.25%",
      "position": "absolute",
      "top": 30,
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "backgroundColor": "rgb(149, 198, 100)",
        "color": "rgb(255, 255, 255)",
        "fontFamily": "Poppins",
        "fontSize": 12,
        "fontWeight": "bold",
        "paddingLeft": 4,
        "paddingRight": 4,
      }
    }
  >
    NO NETWORK OVERRIDE
  </Text>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "height": 38,
        "opacity": 1,
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 10,
        "width": 38,
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/app/components/header/images/settings-icon.png",
        }
      }
      style={
        Object {
          "height": 18,
          "width": 18,
        }
      }
    />
  </View>
</View>
`;
